packages/daemon/src/service/pipeline.ts

```diff
@@ -22,6 +24,14 @@ function getIdOrName(id: string): QueryParams {
   return validate(id) as boolean ? { id } : { name: id };
 }
 
+function execAsync(cmd: string, opts: ExecOptions): Promise<string> {
+  return new Promise((resolve, reject): void => {
+    exec(cmd, opts, (err: ExecException, stdout: string, stderr: string) => {
+      err == null ? resolve(stdout) : reject(err);
```

Comment:
how about: err ? resolve(...) : reject(...)

Reply:
This is an incorrect expression, we should not resolve when there is no errors.

Reply:
Awkward:(, I mean !err ? resolve(...) : reject(...)

Reply:
I don't think it is necessary, you can explain what is wrong with the current way.

Reply:
Nothing else~

Reply:
Resolved, thank you :)